README.md

```diff
@@ -5,19 +5,15 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/mattermost/mattermost-plugin-apps)](https://goreportcard.com/report/github.com/mattermost/mattermost-plugin-apps)
 
 
-# Proof Of Concept - Mattermost Apps
+# Mattermost Apps
 
-This plugin is being developed to test some concepts of creating Apps, which do not rely on a Go executable being installed on the Mattermost server/cluster to extend functionality.  The Apps will not be able to use Go RPC to communicate with the Mattermost Server, only through the "App Plugin" which acts as a sort of proxy to the server's activity.
+This plugin was designed to some concepts of creating Apps, which do not rely on a Go executable being installed on the Mattermost server/cluster to extend functionality. The Apps created using this plugin are not able to use Go RPC to communicate with the Mattermost Server, only through the "App Plugin" which acts as a sort of proxy to the server's activity.
```

Comment:
> The Apps created using this plugin are not able to use Go RPC to communicate with the Mattermost Server, only through the "App Plugin" which acts as a sort of proxy to the server's activity.

This given the wrong impression that app don't use the REST API. Maybe we drop the whole sentence as it weird to say in an introduction what apps are _not_.

Reply:
I really like this explanation and it will work well in the docs - thank you!